example/test/widget_test.dart

```diff
@@ -0,0 +1,31 @@
+// This is a basic Flutter widget test.
```

Comment:
should remove this as this is not a valid test for the example

Reply:
Deleted!